src/core/qgserror.h

```diff
@@ -128,6 +128,12 @@ class CORE_EXPORT QgsError
     //! Clear error messages
     void clear() { mMessageList.clear(); }
 
+    /**
+     * \brief messageList return the list of current error messages
+     * \return current list of error messages
+     */
+    const QList<QgsErrorMessage> messageList() { return mMessageList; }
```

Comment:
That const here should be moved to the end of the signature.

Reply:
Are you sure that we don't want to "add" a const after the signature? 
I wanted the getter to return a const list too, to feed the range loops.

Reply:
Yes, the idea was to add one after the signature. And to remove the one before, but that's already discussed above.